### ISSUE TITLE
Quickly clear all tpys

### ISSUE THREAD

**Issue by osvein** [Ano-Tech-Computers/TelePlugin#5]
Users should be able to quickly tpn everyone by one single command.

Perhaps `/tpn *`?

**Comment by osvein** [comment]
There's a difference between _permission denied_ and _permission not granted_. I think we should discuss whether such a command would _deny everyone_ or _revoke all granted permissions_.

**Comment by osvein** [comment]
As per my understanding, denying someone blocks them from even sending a request. 

As a player, I'd like to be able to make sure no one I have granted permission before suddenly teleport to me. However, I'd like to receive and review their request. Therefore I think such a command should revoke all granted permissions rather than denying everyone.

That being said, I also think the third way to respond (or more specifically _not respond_) to requests, _ignorance_, should be emphasized, as that is more likely to be what the request recipient wants to use, rather than denial.

**Comment by osvein** [comment]
This issue needs clarification through discussion. Hence the _invalid_ and _question_ labels.

**Comment by FloydATC** [comment]
You are correct, the presence of a denial token (pre-emptive or as a response) means any further requests will be ignored. This is to prevent an offensive player from spamming requests.

My suggestion is that "/tpn *" is implemented as a quick way to delete any permission tokens for the issuing player, and (corresponding) "/tpy *" is implemented as a quick way to delete any denial tokens for the issuing player.
This would effectively reset to default and any further requests (/tpa) would be shown.

To clarify, "/tpn *" should NOT be interpreted as "deny and squelch everyone", just as "/tpy *" should NOT be interpreted as "allow everyone to tp freely". It should simply mean "forget I said yes/no, let me know next time someone asks".

**Comment by osvein** [comment]
How about a `--clear` or `--reverse` option that can be applied to `/tpa` as well as `/tpy` and `/tpn`, to reverse an effect (i.e. deleting a token instead of creating it)? Or simply prefixing the player argument with `-` or `!`? Either way makes the intention is a bit clearer, and adds the ability to delete tokens for a specific player as well as everyone.

**Comment by FloydATC** [comment]
I would vote for --clear then. Anyway, is this really a feature that's needed? All tickets are automatically cleared after 60 minutes anyway. What is the actual use case?

**Comment by osvein** [comment]
- Avoid surprise teleport (I might be AFK) when a player accepts an old request.
- Undo incorrectly typed names.

Are you sure it's 60 minutes?

``` java
private final long cooldown = 86400 * 1000; // Milliseconds
```

**Comment by FloydATC** [comment]
Sorry. 24 hours. My mistake.
